tests/admin_widgets/tests.py

```diff
@@ -898,6 +899,54 @@ class DateTimePickerAltTimezoneSeleniumIETests(DateTimePickerAltTimezoneSelenium
 
 
 @override_settings(PASSWORD_HASHERS=['django.contrib.auth.hashers.SHA1PasswordHasher'],
+    ROOT_URLCONF='admin_widgets.urls',
+    TIME_ZONE='America/Chicago',
```

Comment:
I thought about this when writing the patch and decided it makes sense to include it in case the timezone for the admin tests change at some point. The incorrect warnings appear as soon as the client browser is not in UTC (and thus has a localOffset), so this line is actually not really useful and I should remove it.

It would make more sense to set the webdrivers timezone, but I couldn't find an easy way to do that without including external JavaScript mock libraries.

It seems like the current test would return a false positive if the webdrivers profile is using the UTC timezone.